cs_tools/sync/postgres/syncer.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import pydantic
+import sqlalchemy as sa
+import sqlmodel
+from cs_tools.sync.base import DatabaseSyncer
+from typing import TYPE_CHECKING, Any, Literal ,Optional
+from pydantic_core import PydanticCustomError
+import pathlib
+import logging
+from sqlalchemy.orm import sessionmaker
+from cs_tools import __version__
+from cs_tools.sync import utils as sync_utils
+from cs_tools.sync.base import DatabaseSyncer
+
+# if TYPE_CHECKING:
+from cs_tools.sync.types import TableRows
+
+log = logging.getLogger(__name__)
+
+
+class Postgres(DatabaseSyncer):
+    """
+    Interact with Postgres Database
+    """
+    
+    __manifest_path__ = pathlib.Path(__file__).parent / "MANIFEST.json"
+    __syncer_name__ = "postgres"
+   
+    dbname: str 
+    user:  str 
+    password:  str 
```

Comment:
Can we match the other syncers on these names?

- database
- username
- secret